packages/x-components/src/x-modules/scroll/components/main-scroll.vue

```diff
@@ -112,7 +112,7 @@
        * @internal
        */
       const disableAnimations = computed((): boolean => !!pendingScrollTo.value);
-      provide(DISABLE_ANIMATIONS_KEY as string, disableAnimations);
+      provide(DISABLE_ANIMATIONS_KEY as string, disableAnimations.value);
```

Comment:
We are losing the reactivity here leading to side effects. We need to disable the animation while restoring the scroll position from the url but enable them again afterwards. This approach will not work in that case.